pkgs/desktops/xfce/panel-plugins/xfce4-namebar-plugin.nix

```
@@ -2,13 +2,13 @@
 , gtk2, libwnck, libxfce4util, xfce4-panel, wafHook }:
 
 stdenv.mkDerivation rec {
-  ver = "0.3.1";
+  pname = "xfce4-namebar-plugin";
+  version = "0.3.1";
   rev = "07a23b3";
-  name = "xfce4-namebar-plugin-${ver}";
 
   src = fetchFromGitHub {
     owner = "TiZ-EX1";
```

https://github.com/TiZ-EX1/xfce4-namebar-plugin redirects to https://github.com/HugLifeTiZ/xfce4-namebar-plugin

```suggestion
    owner = "HugLifeTiZ";
```

---

Done.